k8s/0.21.6/0.21.6-local-distroless.yaml

```diff
@@ -909,43 +909,51 @@ spec:
             - name: SPIFFE_ENDPOINT_SOCKET
```

Comment:
Things done:
moved all env vars to subcharts and also ordered them alphabetically.